src/app/helpers/app-initializer/app-initializer.ts

```diff
@@ -38,13 +38,11 @@ export class AppInitializer {
  * App values
  */
 export interface Values {
```

Comment:
Why do we have two interfaces?

Reply:
Maybe I'm blind but I can't see where the `Values` interface is used within the `Environment` interface

Reply:
Its used in the `Configuration` interface